app/src/main/java/com/termux/app/ExtraKeysView.java

```diff
@@ -71,9 +75,15 @@ static void sendKey(View view, String keyName) {
             case "↓":
                 keyCode = KeyEvent.KEYCODE_DPAD_DOWN;
                 break;
+            case "↲":
+                keyCode = KeyEvent.KEYCODE_ENTER;
+                break;
             case "―":
                 chars = "-";
                 break;
+            case "-":
+                chars = "-";
+                break;
```

Comment:
What if the guy wrote "\u2015" in the termux properties? We should remove that "case", if he wrote.

The default config should contains the ascii dash (U+002D - HYPHEN-MINUS), but should be *displayed* as "−" (U+2212 − MINUS SIGN) in the shell. See my `display` parameter in the PR.